virttest/utils_misc.py

```diff
@@ -3470,6 +3470,30 @@ def check_device_driver(pci_id, driver_type):
     return driver == driver_type
 
 
+def get_guest_os_datetime(vm, os_type):
```

Comment:
In the __doc__ of the function , it "returns the seconds since the epoch", I think int type is enough, may be the name is confusing as you said, but I think that's ok.  Any suggestions about the function name?